modules/block/README.md

```diff
@@ -1,10 +1,13 @@
 # Block
 
-- Genesis classes and some generated datamodels serializers and deserializers
-- Serializers necessary to serialize Rust specific models to Kotlin models
-- Deserializers necessary to deserialize models to json form which Iroha2 expects for [genesis.json](../testcontainers/src/main/resources/genesis.json) for example.
-  <br/>You can use iroha2Mapper from [deserializers.kt](./src/main/kotlin/jp/co/soramitsu/iroha2/deserializers.kt) for it.
-
-  ```kotlin
-  import jp.co.soramitsu.iroha2.iroha2Mapper
-  ```
\ No newline at end of file
+The `block` module contains:
+
+- Genesis classes and some generated serializers and deserializers for data models
+- Serializers necessary to serialise Rust-specific models to Kotlin models
+- Deserializers necessary to deserialise models into JSON format, which is the format Iroha2 expects, for example, for [genesis.json](../testcontainers/src/main/resources/genesis.json)
+
+Note that you can also use `iroha2Mapper` from [deserializers.kt](./src/main/kotlin/jp/co/soramitsu/iroha2/deserializers.kt) <!-- The link is broken --> to deserialise models into JSON:
```

Comment:
The link is broken, and I couldn't find `iroha2Mapper` anywhere in this repo so I'm not sure where to link to

Reply:
@gv-timur